### PR TITLE
Update baggers.js

### PR DESCRIPTION
**Pull request by ndeloof**

### CODE REVIEW

File: baggers.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-var data = {
+dvar data = {
```
Comment (dadoonet) [inline baggers.js:1]: What is that?

File: baggers.js [hunk 2]
```diff
@@ -15847,393 +15833,4 @@ var data = {
           lang: ['fr', 'en'],
         }
       ],
-      cities: ['Paris', 'Nantes'],
```
Comment (dadoonet) [inline baggers.js:15850]: Why are you removing this part?

File: baggers.js [hunk 3]
```diff
@@ -15847,393 +15833,4 @@ var data = {
...
-        }
-      ],
-      cities: ['Lille', 'Paris', 'Nantes'],
-      contacts: {
-        twitter: 'hsablonniere',
-        mail: 'hubert.sablonniere+bbl@gmail.com',
-      },
-    },
-  ],
-  cities: {
```
Comment (dadoonet) [inline baggers.js:15985]: We do need cities...

**Review by dadoonet — CHANGES_REQUESTED** [review]

File: baggers.js (hunk 1 above)
Comment (ndeloof) [reply baggers.js:1]: a stupid typo

File: baggers.js (hunk 3 above)
Comment (ndeloof) [reply baggers.js:15985]: ok, so Github web editor is not the best tool for this :-\

Comment (dadoonet) [reply baggers.js:15985]: Yeah. I suspect this is because we have a toooooo big file here.

**Pull request closed without merging**